pkgs/tools/misc/xfel/default.nix

```diff
@@ -0,0 +1,43 @@
+{ fetchFromGitHub
+, lib
+, libusb1
+, pkg-config
+, stdenv
+}:
+
+stdenv.mkDerivation rec {
+  pname = "xfel";
+  version = "1.2.9";
+
+  src = fetchFromGitHub {
+    owner = "xboot";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-AvQORR9JPmahHv+pgQxr8Zm8DaaqwURqU746+Qs/Qz8=";
+  };
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ libusb1 ];
+
+  installPhase = ''
+    runHook preInstall
+
+    substituteInPlace 99-xfel.rules \
+      --replace 'GROUP="users"' 'GROUP="felusers"' \
+      --replace 'MODE="666"' 'MODE="660"'
+
+    install -Dm755 xfel $out/bin/xfel
+    install -Dm644 99-xfel.rules $out/lib/udev/rules.d/xfel.rules
+    install -Dm644 LICENSE $out/share/licenses/LICENSES
```

Comment:
Why singular source, but plural destination?

Reply:
Fixed the filename but I guess the directory name is kind of a standard path.